content/blog/2022-03-17-march-22-heartbeat.md

```diff
@@ -37,6 +37,24 @@ tags:
   - Git
 ---
 
+<admon title="Unique Warning Title" type="warn">
```

Comment:
Put down some tests in the latest blog post for review. This pr will be a draft until this testing code is deleted :)